README.md

```diff
@@ -180,32 +180,4 @@ const ComponentWithErrorBoundary = withErrorBoundary(ExampleComponent, {
 
 ---
 
-# FAQ
-## `ErrorBoundary` cannot be used as a JSX component
-This error can be caused by a version mismatch between [react](https://npmjs.com/package/react) and [@types/react](https://npmjs.com/package/@types/react). To fix this, ensure that both match exactly, e.g.:
-
-If using NPM:
-```json
-{
-  ...
-  "overrides": {
-    "@types/react": "17.0.60"
-  },
-  ...
-}
-```
-
-If using Yarn:
-```json
-{
-  ...
-  "resolutions": {
-    "@types/react": "17.0.60"
-  },
-  ...
-}
-```
-
----
-
```

Comment:
I think this is still worth keeping for now, as this _does_ seem to be another potential issue based on @fh-igor-serko's comment.